types/selenium-standalone/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es2019"
```

Comment:
why this is required?

Reply:
Not at all.
The project has dropped NodeJS 10 support and now the minimum required is 12 that supports `es2019`.

Safe to remove if you prefer

Reply:
Just verified, lint and test passing with `es6`.
Shall I revert this change?

Reply:
Yes, I'd say so. My concern was DT package interop with other packages, nothing specific.

Reply:
Ok